department-of-veterans-affairs/va.gov-cms
# VACMS-2263: Add faq and connect with us panels and set spotlight default paragraph to none.

## Description
See #2263 #2264 

## Testing done
Visual / behat

## Screenshots
![Screenshot from 2020-12-08 13-54-57](https://user-images.githubusercontent.com/2404547/101528232-0b582580-395d-11eb-8ede-d6d1f5bf7878.png)
![Screenshot from 2020-12-08 13-55-11](https://user-images.githubusercontent.com/2404547/101528233-0bf0bc00-395d-11eb-9f1e-26c5a68b0d3d.png)
![Screenshot from 2020-12-08 15-23-13](https://user-images.githubusercontent.com/2404547/101537394-642dbb00-3969-11eb-9af2-07c1d0df9afb.png)

## QA steps
As an administrator, go to `/node/add/campaign_landing_page` and create all necessary panels, with the exception of FAQ. Confirm saving doesn't throw faq errors.
After saving, visually verify that faq has boolean toogle that hides / shows fields as expected.
Add 2 qa's to faq, save and confirm limit error is thrown.
Add 11 qa's to faq, save and confirm limit error is thrown.
Add 3 qa's to faq, save and confirm no limit error is thrown.
Visually verify that `Connect with us panel` appears as in last screenshot

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
